### ISSUE TITLE
HTTP 500 by Accept header from Chrome 79

### ISSUE THREAD

**Issue by mathias-goebel** [eXist-db/exist#3173]
As reported in #2679 the current Accept header sent by Google Chrome forces a HTTP 500, again.
> eXist 5.1.0-SNAPSHOT 

### Reproduce
```bash
docker run -p 8099:8080 existdb/existdb:latest & sleep 30;
while [ $(curl --head --silent http://localhost:8099/ --connect-timeout 5 | grep -c "200 OK") = 0 ]; do sleep 2s; done;
echo 'xquery version "3.1";
module namespace api = "http://test.org/ns/exist/api";
declare namespace output = "http://www.w3.org/2010/xslt-xquery-serialization";
declare namespace rest = "http://exquery.org/ns/restxq";

(:~
 : Test
 :)
declare
  %rest:GET
  %rest:path("/info")
  %rest:produces("application/json")
  %output:media-type("application/json")
  %output:method("json")
function api:info()
as map(*) {
  map {
    "name": "test"
  }
};' > /tmp/test.xqm
curl -X PUT -H 'Content-Type: application/xquery' --data-binary @/tmp/test.xqm http://admin:@localhost:8099/exist/rest/db/test.xqm
# google-chrome http://localhost:8099/exist/restxq/info
# or in the terminal:
curl -H "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9" http://localhost:8099/exist/restxq/info
```
### Error Message
```
HTTP ERROR 500 javax.servlet.ServletException: javax.servlet.ServletException:
An error occurred while processing request to /exist/restxq/info: Invalid Accept Header Value: &apos;text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9&apos; in respect to pattern: &apos;((?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)(,\s?(?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)*&apos;</h2>
<table>
<tr><th>URI:</th><td>/exist/restxq/info</td></tr>
<tr><th>STATUS:</th><td>500</td></tr>
<tr><th>MESSAGE:</th><td>javax.servlet.ServletException: javax.servlet.ServletException: An error occurred while processing request to /exist/restxq/info: Invalid Accept Header Value: &apos;text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9&apos; in respect to pattern: &apos;((?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)(,\s?(?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)*&apos;</td></tr>
<tr><th>SERVLET:</th><td>XQueryURLRewrite</td></tr>
<tr><th>CAUSED BY:</th><td>javax.servlet.ServletException: javax.servlet.ServletException: An error occurred while processing request to /exist/restxq/info: Invalid Accept Header Value: &apos;text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9&apos; in respect to pattern: &apos;((?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)(,\s?(?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)*&apos;</td></tr>
<tr><th>CAUSED BY:</th><td>javax.servlet.ServletException: An error occurred while processing request to /exist/restxq/info: Invalid Accept Header Value: &apos;text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9&apos; in respect to pattern: &apos;((?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)(,\s?(?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)*&apos;</td></tr>
<tr><th>CAUSED BY:</th><td>java.lang.IllegalArgumentException: Invalid Accept Header Value: &apos;text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9&apos; in respect to pattern: &apos;((?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)(,\s?(?:(?:\*\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/\*)|(?:[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}\/[a-z0-9!#\$&amp;\.\+\-\^_]{1,127}))(?:;\s?q=(?:0(?:\.[0-9]{1,3})?)|(?:1(?:\.[0]{1,3})?))?\s?(?:;\s?[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+=[!#$%&amp;&apos;*+\-.^_`|~0-9a-zA-z]+|(?:&quot;[a-z0-9]+&quot;))?)*&apos;</td></tr>
</table>
	[…]
```

**Comment by dizzzz** [comment]
@mathias-goebel please add ass requested all information from the template; e.g. you do not mention the eXist-db  version :-(

**Comment by adamretter** [comment]
I just released version 0.1.35 of the EXQuery RESTXQ implementation, in which I wrote a new parser for Accept Headers. I think that should solve the problem. Hopefully @dependabot-bot will pick up the latest release for us in the next 24 hours.

**Comment by adamretter** [comment]
Closed by https://github.com/eXist-db/exist/pull/3198
